Pokedex_ios/View/Cells/GenerationCell/GenerationCollectionViewCell.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  GenerationCollectionViewCell.swift
+//  Pokedex_ios
+//
+//  Created by admin on 7/12/22.
+//
+
+import UIKit
+
+class GenerationCollectionViewCell: UICollectionViewCell {
+
+    @IBOutlet weak var generationImageView: UIImageView!
+    @IBOutlet weak var generationLabel: UILabel!
+    @IBOutlet weak var generationBackground: UIView!
+    @IBOutlet weak var patternImageView: UIImageView!
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+    }
+
+    func setupData(generation: GenerationFilter, isTypeSelected: Bool) {
+        generationLabel.text = generation.name
+        let imageName = "pokemon-\(generation.description)"
+        generationImageView.image = UIImage(named: imageName)
+
+        if isTypeSelected {
+            generationLabel.textColor = UIColor(named: "TextWhiteColor")
```

Comment:
These colors have already been declared on other parts of the app, please reuse them